SerialQueue/SerialQueue/SerialQueue.cs

```diff
@@ -5,12 +5,13 @@ namespace Threading
 {
     public class SerialQueue
     {
-        readonly object _locker = new object();
-        WeakReference<Task> _lastTask;
+        private readonly object _locker = new object();
+        private readonly WeakReference<Task> _lastTask = new WeakReference<Task>(null);
```

Comment:
Please let's keep current code style without private keyword.

Reply:
Thanks for the review. Will change my code to comply. Sorry for delayed response